boards/riscv/qemu_riscv32/qemu_riscv32.yaml

```diff
@@ -6,6 +6,7 @@ arch: riscv32
 ram: 16
 toolchain:
   - zephyr
+  - gnuriscvemb
```

Comment:
Worth mentioning from toolchain_3rd_party_x_compilers.rst? @tejlmand

Reply:
Thanks, I'll check there and see about adding documentation elsewhere as well.

Reply:
doc/guides/c_library.rst might also be a good place to mention the new Kconfig option, since that touches on how to use newlib.

Reply:
Thanks for the pointers - reviewing these docs, I discovered that what I want is the CROSS_COMPILE environment variable instead of gnuriscvemb. I think I should just remove that patch from the series.